src/last_id_queue.rs

```diff
@@ -52,6 +52,19 @@ impl LastIdQueue {
     pub fn check_entry(&self, entry_id: Hash) -> bool {
         self.entries.get(&entry_id).is_some()
     }
+
+    pub fn genesis_last_id(&mut self, last_id: &Hash) {
```

Comment:
Sweet.  I wasn't bold enough to add a function that looked like this when I did my initial work here